Docker/FreshRSS.Apache.conf

```diff
@@ -53,7 +57,6 @@ CustomLog "|/var/www/FreshRSS/cli/sensitive-log.sh" combined_proxy
 		OIDCXForwardedHeaders ${OIDC_X_FORWARDED_HEADERS}
 	</IfDefine>
 
-	# Can be overridden e.g. in /var/www/FreshRSS/p/i/.htaccess
```

Comment:
Out of curiosity: isn't it working?

Reply:
Ouch, nice caught! I don't know what I did sorry. Fixed in 4f697d37bfa15a1b189d4765ddf1c5e29f314680

Reply:
If you have a chance to test that it actually works, that would be nice. I do not have an OIDC setup at the moment, to experiment